gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/ConsultationListMapperTest.java

```diff
@@ -80,23 +82,29 @@ public void testValidFullDataConsultationList() {
         var ehrExtract = unmarshallEhrExtractElement(FULL_VALID_CONSULTATION_LIST_XML);
         setUpEncounter("20100113152000", "20150213152000", "test-display", "test-text");
 
-        var consultation = listMapper.mapToConsultation(ehrExtract, encounter);
+        var comp = ehrExtract.getComponent().get(0).getEhrFolder().
+                getComponent().get(0).getEhrComposition();
+        var consultation = listMapper.mapToConsultation(comp, encounter);
 
-        assertConsultation(consultation, "20100113152000", "test-text");
+        assertConsultation(consultation, "20100113151332", "test-text");
     }
 
     @Test
     public void testValidNoOptionalDataConsultationList() {
```

Comment:
@azam145 Did you test drive this new functionality at all? I can't see any tests you've added for this new functionality.

Reply:
Added further tests